src/server/listener.cc

```diff
@@ -145,6 +147,27 @@ Listener::pinWorker(size_t worker, const CpuSet& set)
 #endif
 }
 
+bool
+Listener::systemSupportsIpv6(){
+    struct ifaddrs *ifaddr, *ifa;
+    int family, n;
+    bool supportsIpv6 = false;
+    if (getifaddrs(&ifaddr) == -1) {
+        throw std::runtime_error("Call to getifaddrs() failed");
+    }
+    
+    for (ifa = ifaddr, n = 0; ifa != NULL; ifa = ifa->ifa_next, n++) {
+            if (ifa->ifa_addr == NULL)
+                continue;
+
+            family = ifa->ifa_addr->sa_family;
+            if (family == AF_INET6)
+                continue;
+    }
+    freeifaddrs(ifaddr);
+    return supportsIpv6;
```

Comment:
You never actually set supportsIpv6 to true, and that's the most important part!

Reply:
Doh! Looks like that got dropped during some minor refactoring before submission.
Fixed now.